jdaviz/configs/default/tests/test_aida.py

```diff
@@ -26,7 +26,7 @@ def test_get_viewport_sky(imviz_helper, image_hdu_wcs):
     viewport = viewer.aid.get_viewport('sky')
 
     expected_center = SkyCoord(ra=337.51894337, dec=-20.83208305, unit='deg')
-    expected_fov = 0.00277778 * u.deg
+    expected_fov = 0.00277778 * u.arcsec
```

Comment:
Is this right? That's an incredibly tiny FOV.

Reply:
great eye! no idea when this changed, but I'm gonna change it back